<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIRBodyDialog</class>
 <widget class="QWidget" name="GUIRBodyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rigid Body</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Inertia:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Rigid Body Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Sensor property:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_MainNodeID"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Inertia Jxx:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Secondary nodes group ID:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Radioss HMPP:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Main Node ID:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Inertia Jzz:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Rigid Body KeyWord:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Center of gravity computation:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_MainNodeID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Mass:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Inertia Jxz:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Surface ID:</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Inertia Jxy:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Inertia Jyy:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Skew:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Rigid wall deactivation:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Inertia Jyz:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_Auto">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_KeyWord"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Name"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_sens_ID"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_Skew_ID"/>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_Ispher"/>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Mass"/>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="GUI::NodeGroupWidget" name="widget_SecNode" native="true"/>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_Ikrem"/>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_ICoG"/>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="GUI::NodeGroupWidget" name="widget_Surf" native="true"/>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Jxx"/>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Jyy"/>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Jzz"/>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Jxy"/>
   </item>
   <item row="15" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Jyz"/>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Jxz"/>
   </item>
   <item row="17" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_Ioptoff"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::NodeGroupWidget</class>
   <extends>QWidget</extends>
   <header>NodeGroupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
